regtests/bin/matrix.base

```diff
@@ -2152,16 +2152,6 @@
           echo "./bin/test.comp ww3_ufs1.1 work_a work_b"                                                  >> matrix.body
       fi
      fi
-
```

Comment:
Was this intentionally taken out?  I think we'd like to keep this regression test in.